internal/danger/danger.go

```diff
@@ -17,21 +16,19 @@ func SubsliceOffset(data []byte, subslice []byte) int {
 	}
 	offset := hlp.Data - datap.Data
 
-	if offset > maxInt {
+	if offset > math.MaxInt {
```

Comment:
`math.MaxInt` only exists in Go 1.17+. Do you mind moving the current definition of maxInt in the `sizetype.go` file?

Officially go-toml doesn't support versions older than the last two Go versions, but I feel like in this case it would be a small effort to keep the compatibility for older versions.

Reply:
> `math.MaxInt` only exists in Go 1.17+. Do you mind moving the current definition of maxInt in the `sizetype.go` file?

Sounds good to define privately. It's probably a better idea to do it in `danger.go` itself, to not define it twice in `sizetype.gp` and `sizetype_tinygo.go`.